### PR TITLE
Reformat conv1d_transpose

### PR DESCRIPTION
**Pull request by xoiga123**

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by xoiga123** [comment]
@agniiyer Hi, can you review this please?

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

File: ivy/array/layers.py [hunk 1]
```diff
@@ -372,15 +372,57 @@ def conv1d(
...
+        output_shape
+            Shape of the output (Default value = None)
+        data_format
+            "NWC" or "NCW". Defaults to "NWC".
+        dilations
+            The dilation factor for each dimension of input. (Default value = 1)
+        out
+            optional output array, for writing the result to. It must have a shape that
+            the inputs broadcast to.
+
```
Comment (AnnaTz) [inline ivy/array/layers.py:409]: The parameters description should match the one in the current master.

File: ivy/container/layers.py [hunk 2]
```diff
@@ -1125,19 +1125,73 @@ def conv2d(
...
+        prune_unapplied
+            Whether to prune key_chains for which the function was not applied.
+            Default is ``False``.
+        map_sequences
+            Whether to also map method to sequences (lists, tuples).
+            Default is ``False``.
+        out
+            optional output container, for writing the result to. It must have a shape
+            that the inputs broadcast to.
+
```
Comment (AnnaTz) [inline ivy/container/layers.py:1177]: Likewise here.

File: ivy/functional/ivy/layers.py [hunk 3]
```diff
@@ -916,6 +916,66 @@ def conv1d_transpose(
     ret
         The result of the transpose convolution operation.
 
+    Both the description and the type hints above assumes an array input for simplicity,
+    but this function is *nestable*, and therefore also accepts :class:`ivy.Container`
+    instances in place of any of the arguments.
```
Comment (AnnaTz) [inline ivy/functional/ivy/layers.py:921]: This is only to be mentioned in the container script's docstrings.

File: ivy/functional/ivy/layers.py [hunk 4]
```diff
@@ -916,6 +916,66 @@ def conv1d_transpose(
...
+    {
+        a: {
+            c: [1,28,6],
+            d: [1,28,6]
+        },
+        b: {
+            c: [1,56,6],
+            d: [1,56,6]
+        }
+    }
```
Comment (AnnaTz) [inline ivy/functional/ivy/layers.py:978]: It would be nice to also include a :class:`ivy.NativeArray` example here.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/layers.py (hunk 3 above)
Comment (xoiga123) [reply ivy/functional/ivy/layers.py:921]: I believe this is added in ivy's functional API though? The other functions and the docs don't mention it in a container's docstrings.

**Comment by xoiga123** [comment]
@AnnaTz Great job reformatting the convolutional functions! I just wanted to ask how you knew I was working on `conv1d_transpose` and left the type hints unformatted on purpose, so that I don't have to close this PR?

File: ivy/functional/ivy/layers.py (hunk 3 above)
Comment (AnnaTz) [reply ivy/functional/ivy/layers.py:928]: Yes, you're right, my bad.

**Comment by AnnaTz** [comment]
> @AnnaTz Great job reformatting the convolutional functions! I just wanted to ask how you knew I was working on `conv1d_transpose` and left the type hints unformatted on purpose, so that I don't have to close this PR?

It wasn't on purpose tbh 😅 While refactoring the convolution functions I was mainly interested in getting the parameter description right in the main ivy script. I didn't pay that much attention to the type hints it seems 😏

**Pull request merged**